cli/pkg/local/app.go

```diff
@@ -47,6 +47,7 @@ const (
 	DefaultInstanceID = "default"
 	DefaultOLAPDriver = "duckdb"
 	DefaultOLAPDSN    = "stage.db"
```

Comment:
Now that we're anyway moving it, can we change it to `main.db`? That would also fit with the default DuckDB schema name